### ISSUE TITLE
11 productionish server.md

### ISSUE THREAD

**Issue by laeg** [reactjs/react-router-tutorial#33]
NODE_ENV=production npm start

Shouldn't this be:

env NODE_ENV=production npm start

**Comment by bartimaeus** [comment]
@laeg I don't think so. What you are doing by calling `NODE_ENV=production` without `env` is that you set the temporary environment variable for your current session without running a new process in the background.

Here's a better explanation from the Ubuntu forum (http://askubuntu.com/a/205698):

`env`, on another hand, is a program in itself. When `env` is called, [the] following things happen:

* The command `env` gets executed as a new process
* `env` modifies the environment, and
* calls the command that was provided as an argument. The `env` process is replaced by the `command` process.

Example:

> env GREP_OPTIONS='-v' grep one test.txt

This command will launch two new processes: (i) env and (ii) grep (actually, the second process will replace the first one). From the point of view of the grep process, the result is exactly the same as running

> GREP_OPTIONS='-v' grep one test.txt

However, you can use this idiom if you are outside of bash or don't want to launch another shell (for example, when you are using the `exec()` family of functions rather than the `system()` call).

**Comment by stereoit** [comment]
`env` is needed when using other shell like fishshell.com
